source/jormungandr/jormungandr/interfaces/v1/serializer/geocode_json.py

```diff
@@ -167,6 +167,45 @@ class PoiTypeSerializer(serpy.DictSerializer):
     name = serpy.StrField(display_none=True)
 
 
+class PoisSerializer(serpy.Field):
+    def as_getter(self, serializer_field_name, serializer_cls):
+        return lambda obj: self.make(obj)
+
+    def make(self, obj):
+        children = value_by_path(obj, 'properties.geocoding.children', [])
+        if not children:
+            return None
+
+        def make_children(child):
```

Comment:
It creates only one child.
```suggestion
        def make_child(child):
```

Reply:
Done